src/PgRvn.Server/Types/PgBit.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using PgRvn.Server.Exceptions;
+using PgRvn.Server.Messages;
+
+namespace PgRvn.Server.Types
+{
+    public class PgBit : PgType
+    {
+        public static readonly PgBit Default = new();
+        public override int Oid => PgTypeOIDs.Bit;
+        public override short Size => sizeof(byte);
+        public override int TypeModifier => 1;
+
+        public override byte[] ToBytes(object value, PgFormat formatCode)
```

Comment:
We probably should avoid `byte[]`, it will force us to allocate, need to think about another interface.

Reply:
Changed to ReadOnlyMemory<byte>